test/test_sparse_ops.py

```diff
@@ -0,0 +1,367 @@
+import math
+from itertools import product, chain
+from numbers import Number
+import random
+
+import unittest
+
+import torch
+
+from torch.testing._internal.common_utils import \
+    (TestCase, run_tests, torch_to_numpy_dtype_dict, suppress_warnings,
+     TEST_NUMPY, IS_MACOS, make_tensor)
+from torch.testing._internal.common_methods_invocations import \
+    (sparse_unary_ufuncs)
+from torch.testing._internal.common_device_type import \
+    (instantiate_device_type_tests, ops, dtypes)
+from torch.testing import \
+    (floating_types_and, integral_types, all_types_and_complex_and)
+
+if TEST_NUMPY:
+    import numpy as np
+
+# Functions tested here:
+#
+
+# Interesting values and extremal values for different dtypes
```

Comment:
This doesn't need to create a new test suite. Use the existing sparse test suite but add a class. Start with just one test. The test runs for every OpInfo that supports sparse. This test should create some sparse tensors, call the op on them, and then compare the result to the same operation called on the equivalent dense tensors.

Reply:
Made the changes as per the suggestion